src/App.jsx

```diff
@@ -3,7 +3,7 @@ import './App.css'
 import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
 
-function useSemiPersistentState() {
+const useSemiPersistentState = () => {
   
   const [todoList, setTodoList] = useState(() => {
     const savedTodoList = localStorage.getItem('savedTodoList')
```

Comment:
@vankampenc You missed the part of the code that saves the data back into localstorage